py/picca/fitter2/utils.py

```diff
@@ -0,0 +1,117 @@
+
+import scipy as sp
+from numpy import fft
+from scipy import special
+import scipy.interpolate
+
+from . import myGamma
+
+nmuk = 1000
+muk=(sp.arange(nmuk)+0.5)/nmuk
+dmuk = 1./nmuk
+muk=muk[:,None]
+
+def sinc(x):
+    return sp.sin(x)/x
+
+def Pk2Mp(ar,k,pk,ell_max=None):
+    """
+    Implementation of FFTLog from A.J.S. Hamilton (2000)
+    assumes log(k) are equally spaced
+    """
+
+    k0 = k[0]
+    l=sp.log(k.max()/k0)
+    r0=1.
+
+    N=len(k)
+    emm=N*fft.fftfreq(N)
+    r=r0*sp.exp(-emm*l/N)
+    dr=abs(sp.log(r[1]/r[0]))
+    s=sp.argsort(r)
+    r=r[s]
+
+    xi=sp.zeros([ell_max/2+1,len(ar)])
+
+    for ell in range(0,ell_max+1,2):
+        pk_ell=sp.sum(dmuk*L(muk,ell)*pk,axis=0)*(2*ell+1)*(-1)**(ell/2)
```

Comment:
be careful with `/` versus `//`.
It will not work in Python3

Reply:
What does a simple `/` do in python3 when the operands are integer and the divisor is a factor of the dividend (as is the case here since ell is always even)? Does it cast the result to float? I can change to `//` but I'd prefer the code to break rather than spitting out the wrong result...